src/app/genes/gene-overview/gene-overview.component.ts

```diff
@@ -466,7 +466,7 @@ export class GeneOverviewComponent implements OnInit, OnDestroy, AfterContentChe
     }
 
     getAlias(): string {
-        if (this.geneInfo.alias.length > 0) {
+        if (this.geneInfo.alias && this.geneInfo.alias.length > 0) {
```

Comment:
This is a tiny bug fix that didn't deserve it's own PR, it's entirely unrelated to the rest of this PR...